<launch>


<include file="$(find realsense2_camera)/launch/rs_camera.launch"/>

<node name="QAQ" pkg="cupDetection" type="cv_test" output="screen">
    <remap from="image_raw" to="/camera/color/image_raw"/>
</node>


<!--  for green cup lowHSV = <60,160,0> HighHSV = <100,255,255>-->

<!--  for red cup lowHSV = <0,100,0> HighHSV = <180,255,230>-->
<param name="green_lowH"  type="double" value="60"/>
<param name="green_lowS"  type="double" value="160"/>
<param name="green_lowV"  type="double" value="0"/>
<param name="green_highH" type="double" value="100"/>
<param name="green_highS" type="double" value="255"/>
<param name="green_highV" type="double" value="255"/>


<param name="red_lowH"  type="double" value="10"/>
<param name="red_lowS"  type="double" value="0"/>
<param name="red_lowV"  type="double" value="0"/>
<param name="red_highH" type="double" value="177"/>
<param name="red_highS" type="double" value="255"/>
<param name="red_highV" type="double" value="255"/>

<param name="white_lowR"  type="double" value="150"/>
<param name="white_lowG"  type="double" value="150"/>
<param name="white_lowB"  type="double" value="150"/>
<param name="white_highR" type="double" value="255"/>
<param name="white_highG" type="double" value="255"/>
<param name="white_highB" type="double" value="255"/>


<param name="black_lowR"  type="double" value="0"/>
<param name="black_lowG"  type="double" value="0"/>
<param name="black_lowB"  type="double" value="0"/>
<param name="black_highR" type="double" value="80"/>
<param name="black_highG" type="double" value="80"/>
<param name="black_highB" type="double" value="80"/>

<!-- <param name="xPose" type="int" value="330"/>
<param name="yPose" type="int" value="300"/> -->
<param name="height" type="int" value="480"/>
<param name="width" type="int" value="640"/>
<param name="circleSize" type="int" value="60"/>

</launch>